lib/matrix_check_sum.rb

```diff
@@ -1,10 +1,14 @@
-# Checks that for the given matrix, where number of rows are equal to number of columns
-# whether the sum of each row matches the sum of corresponding column i.e. sum
-# of numbers in row i is the same as the sum of numbers in column i for i = 0 to row.length-1
-# If this is the case, return true. Otherwise, return false.
-
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n) where n is the number of items rows/cols in the matrix
```

Comment:
I _think_ I get what you mean here, but can you clarify what exactly `n` is here? Is it the number of:
- items (rows * columns)
- rows
- columns
- something else?